tests/functional/lib/bash/test_header

```diff
@@ -782,6 +782,13 @@ skip_all() {
     exit
 }
 
+skip_macos_gh_actions() {
+    # https://github.com/cylc/cylc-flow/issues/6276
+    if [[ "$CI" && "$OSTYPE" == "darwin"* ]]; then
```

Comment:
Should this have a default?

Reply:
https://github.com/cylc/cylc-flow/pull/6300